flexprep/domain/prepare_processing.py

```diff
@@ -80,7 +80,7 @@ def launch_pre_processing(self, objects):
 
                 logging.info(f"Launching Pre-Processing for timestep {step}")
                 if prev_step == 0:
-                    Processing().process(step_zero + [file])
+                    Processing().process(step_zero + [file.copy()])
```

Comment:
I had to include it because, otherwise, when verifying that the process method:
```
Processing().process(step_zero + [file])
file["processed"] = "Y"
```
was called with the expected arguments, it fails by incorrectly indicating that `file["processed"]` should be `"Y"`, even though this is false when the process function is called.

Reply:
Let's leave this as is until the implementation using sqlite.